src/registrar/admin.py

```diff
@@ -45,7 +45,44 @@ def resource(self, obj):
     add_form_template = "admin/change_form_no_submit.html"
 
 
-class AuditedAdmin(admin.ModelAdmin, AdminSortFields):
+class AdminSortFields:
+    def get_queryset(db_field):
+        """This is a helper function for formfield_for_manytomany and formfield_for_foreignkey"""
```

Comment:
**comment (non-blocking)**: This function has swallowed lots of domain-specific logic by special-casing behavior based on all of our field names. I'm concerned that we won't be able to track down this single function if we change something.

Could we either expand the docstring to warn about this, or add it into our developer documentation, or both?